frontend/lib/identification/no_card_view.dart

```diff
@@ -14,96 +14,117 @@ class NoCardView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final isLandscape = MediaQuery.of(context).orientation == Orientation.landscape;
+    return LayoutBuilder(
+      builder: (BuildContext context, BoxConstraints viewportConstraints) => SingleChildScrollView(
+        child: ConstrainedBox(
+          constraints: BoxConstraints(minHeight: viewportConstraints.maxHeight),
+          child: SafeArea(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                _TapableCardWithArea(
+                  onTap: startApplication,
+                  title: "Beantragen",
+                  description:
+                      "Sie sind ehrenamtlich engagiert, haben aber noch keine Ehrenamtskarte? Hier können Sie Ihre Ehrenamtskarte beantragen.",
+                  icon: Icons.assignment,
+                ),
+                _TapableCardWithArea(
+                  onTap: startActivation,
+                  title: "Karte aktivieren",
+                  description:
+                      "Sie haben die Ehrenamtskarte bereits beantragt und den Aktivierungscode Ihrer digitalen Ehrenamtskarte erhalten? Scannen Sie den Code hier ein.",
+                  icon: Icons.add_card,
+                ),
+                _TapableCardWithArea(
+                  onTap: startVerification,
+                  title: "Gültigkeit prüfen",
+                  description:
+                      "Sie möchten die Gültigkeit einer digitalen Ehrenamtskarte prüfen? Scannen Sie den Code hier ein.",
+                  icon: Icons.verified,
+                ),
+              ].wrapWithSpacers(height: 24),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
 
-    Widget wrapIntrinsic(Widget widget) => isLandscape ? IntrinsicHeight(child: widget) : IntrinsicWidth(child: widget);
+class _TapableCardWithArea extends StatelessWidget {
```

Comment:
We might want to move this to a separate file.

Reply:
I would only do it, once we have usages other than NoCardView